source/styles.tex

```diff
@@ -142,13 +142,13 @@
 %% (copied verbatim from listings.sty version 1.6 except where commented)
 \makeatletter
 
-\lst@CheckVersion{1.8d}{\lst@CheckVersion{1.8c}{\lst@CheckVersion{1.8b}{\lst@CheckVersion{1.7}{\lst@CheckVersion{1.6}{\lst@CheckVersion{1.5b}{
+\lst@CheckVersion{1.9}{\lst@CheckVersion{1.8d}{\lst@CheckVersion{1.8c}{\lst@CheckVersion{1.8b}{\lst@CheckVersion{1.7}{\lst@CheckVersion{1.6}{\lst@CheckVersion{1.5b}{
```

Comment:
What happens if we just don't check this?

Reply:
The problem is I'm overwriting internal macros of the listing package, which may or may not work in a future version.

But I guess that uncertainty is better than a reliable failure with each new version of the package.

Reply:
Let's get rid of the check.